dev/managedservicesplatform/operationdocs/diagram/msp.go

```diff
@@ -0,0 +1,146 @@
+		}
+		// move loadbalancer into the container
+		graph, loadbalancer, err = move(graph, container, loadbalancer, true)
+		if err != nil {
+			return graph, container, errors.Wrap(err, "failed to move ALB into container")
+		}
+
+		graph, cloudarmor, err := createWithIcon(graph, "Cloud Armor", CloudArmor)
+		if err != nil {
+			return graph, container, errors.Wrap(err, "failed to create cloud armor")
+		}
+		// move cloudarmor into the container
+		graph, cloudarmor, err = move(graph, container, cloudarmor, true)
+		if err != nil {
+			return graph, container, errors.Wrap(err, "failed to move cloud armor into container")
+		}
+
+		graph, err = addBidirectionalConnection(graph, loadbalancer, cloudarmor, "")
+		if err != nil {
+			return graph, container, errors.Wrap(err, "failed to add connection between loadbalancer and cloudarmor")
+		}
+		return graph, container, nil
+
+	}
+	return createWithIcon(graph, "Application Load Balancer", CloudLoadBalancer)
+}
+
+func newMonitoringNode(graph *d2graph.Graph, _ *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	return createWithIcon(graph, "Monitoring", CloudMonitoring)
+}
+
+func newOpsgenieNode(graph *d2graph.Graph, _ *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	return createWithIcon(graph, "Opsgenie", Opsgenie)
+}
+
+func newPostgresNode(graph *d2graph.Graph, env *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	graph, key, err := createContainer(graph, "Cloud SQL (Postgres)", CloudSQL)
+	if err != nil {
+		return graph, key, errors.Wrap(err, "failed to create postgres container")
+	}
+	// postgres will be a container so we want to set the position of the label + icon to look better
+	graph, err = d2oracle.Set(graph, nil, key+".label.near", nil, pointers.Ptr("outside-top-center"))
+	if err != nil {
+		return graph, key, errors.Wrap(err, "failed to set postgres label location")
+	}
+	graph, err = d2oracle.Set(graph, nil, key+".icon.near", nil, pointers.Ptr("top-center"))
+	if err != nil {
+		return graph, key, errors.Wrap(err, "failed to set postgres icon location")
+	}
+
+	for _, database := range env.Resources.PostgreSQL.Databases {
+		graph, _, err = d2oracle.Create(graph, nil, key+"."+database)
+		if err != nil {
+			return graph, key, errors.Wrapf(err, "failed to create database %s", database)
+		}
+	}
+
+	return graph, key, nil
+}
+
+func newRedisNode(graph *d2graph.Graph, _ *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	return createWithIcon(graph, "Redis", CloudMemorystore)
+}
+
+func newSentryNode(graph *d2graph.Graph, _ *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	return createWithIcon(graph, "Sentry", Sentry)
+}
+
+func newTraceNode(graph *d2graph.Graph, _ *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	return createWithIcon(graph, "Cloud Trace", CloudTrace)
+}
+
+func newVPCNode(graph *d2graph.Graph, _ *spec.EnvironmentSpec) (*d2graph.Graph, string, error) {
+	return createWithIcon(graph, "VPC", VPC)
```

Comment:
```suggestion
	return createWithIcon(graph, "VPC Network", VPC)
```

following the naming here: https://cloud.google.com/vpc/docs/vpc